### PR TITLE
[LLDP] Fix for LLDP advertisements being sent with wrong information.

### PR DESCRIPTION
**Pull request by shlomibitton**
Signed-off-by: Shlomi Bitton <shlomibi@nvidia.com>

**- Why I did it**
Since lldpd is starting before lldpmgrd, some advertisement packets might sent with default value, mac address as Port ID.
This fix hold the packets from being sent by the lldpd until all interfaces are well configured by the lldpmgrd.

**- How I did it**
* Add 'pause' command when lldpd start to avoid false advertisement information.
* Resume lldp operations after ports init is done on DB and all lldpd interfaces configured with correct data.

**- How to verify it**
Capture LLDP advertisement sent from the switch and observe 'Port ID' field.

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [x] 201911
- [x] 202006

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by shlomibitton** [comment]
retest this please

**Comment by keboliu** [comment]
retest this please

**Comment by liat-grozovik** [comment]
retest this please

File: dockers/docker-lldp/lldpmgrd [hunk 1]
```diff
@@ -219,6 +242,18 @@ def main():
 
     lldpmgr.run()
 
+def resume_lldpd(self):
+    cmd = "lldpcli resume"
+    proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```
Comment (pavel-shirshov) [inline dockers/docker-lldp/lldpmgrd:247]: Please exctract "run_cmd" code in a function. And use the function in both places.

File: dockers/docker-lldp/lldpmgrd [hunk 2]
```diff
@@ -219,6 +242,18 @@ def main():
...
+    (stdout, stderr) = proc.communicate()
+    if proc.returncode != 0:
+        self.log_error("Failed to resume lldpd with command: '{}': {}".format(cmd, stderr))
+        sys.exit(1)
+
+def check_timeout(self, start_time):
+    if time.time() - start_time > PORT_INIT_TIMEOUT:
+        self.log_error("Port init timeout reached ({} seconds), killing lldpmgrd...".format(PORT_INIT_TIMEOUT))
+        sys.exit(1)
```
Comment (pavel-shirshov) [inline dockers/docker-lldp/lldpmgrd:256]: Please Just resume the process to make the behavior as it was before

File: dockers/docker-lldp/lldpmgrd [hunk 3]
```diff
@@ -219,6 +242,18 @@ def main():
 
     lldpmgr.run()
 
+def resume_lldpd(self):
+    cmd = "lldpcli resume"
+    proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    (stdout, stderr) = proc.communicate()
+    if proc.returncode != 0:
+        self.log_error("Failed to resume lldpd with command: '{}': {}".format(cmd, stderr))
+        sys.exit(1)
```
Comment (pavel-shirshov) [inline dockers/docker-lldp/lldpmgrd:251]: Is supervisord going to restart the process right away?

**Review by pavel-shirshov — CHANGES_REQUESTED** [review]
As comments

File: dockers/docker-lldp/lldpmgrd (hunk 1 above)
Comment (shlomibitton) [reply dockers/docker-lldp/lldpmgrd:248]: Fixed, please check

File: dockers/docker-lldp/lldpmgrd (hunk 2 above)
Comment (shlomibitton) [reply dockers/docker-lldp/lldpmgrd]: Fixed, please check

File: dockers/docker-lldp/lldpmgrd (hunk 3 above)
Comment (shlomibitton) [reply dockers/docker-lldp/lldpmgrd]: No, this is the configuration on "/usr/share/sonic/templates/supervisord.conf.j2"

[program:lldpmgrd]
command=/usr/bin/lldpmgrd
priority=5
**autostart=false**
**autorestart=false**
stdout_logfile=syslog
stderr_logfile=syslog
dependent_startup=true
dependent_startup_wait_for=lldp-syncd:running

**Review by pavel-shirshov — APPROVED** [review]

**Review by liat-grozovik — APPROVED** [review]

**Pull request merged**